platform/core/src/utils/combineFrameInstance.ts

```diff
@@ -75,22 +75,24 @@ const combineFrameInstance = (frame, instance) => {
     }
     console.debug('🚀 ~ ImagePositionPatientToUse:', ImagePositionPatientToUse);
 
-    const newInstance = Object.assign(instance, { frameNumber: frameNumber });
+    const newInstance = Object.assign(Object.create(instance), { frameNumber: frameNumber });
```

Comment:
This seems to be fine and is far faster

Reply:
can you remove my console debug above it?